src/TRoom.h

```diff
@@ -277,18 +277,18 @@ inline QDebug operator<<(QDebug debug, const TRoom* room)
         }
```

Comment:
\<aside>How come you never put in code to report the custom exit line colour, whether it ends in an arrow-head and the line style?\</aside>

Reply:
Seems to be an omission

Reply:
See https://github.com/Mudlet/Mudlet/pull/7285